sources/_sector/dex/trades/ronin/_sources.yml

```diff
@@ -1,7 +1,11 @@
 version: 2
 
 sources:
-  - name: katana_dex_ronin
+  - name: katana_v2_dex_ronin
     tables:
       - name: KatanaPair_evt_Swap
-      - name: KatanaFactory_evt_PairCreated
\ No newline at end of file
+      - name: KatanaFactory_evt_PairCreated
```

Comment:
We need to make sure the source here match the name in dune.

Reply:
Thank you so much!
Should be solved now!